### DISCUSSION TITLE
blockcheck - какой приоритет у стратегий?

### DISCUSSION THREAD

**Discussion by testing765** [bol-van/zapret#686] [Q&A]
blockcheck выдает множество доступных вариантов для `tls1.2`
Но в summary выдает вариант с `--dpi-desync=fake`, где нужно еще играться с ttl, который как я понимаю, для разных сайтов, может быть разным, если блочится тспу на разных уровнях
Вот например есть еще вариант с `--dpi-desync=disorder2 --wssize 1:6`, почему он хуже?
Хотя ладно, в readme написано что `wssize может замедлять скорость и/или увеличивать время ответа сайтов`, но а что с другими стратегиями? Не для всех есть инфа в readme
Хотелось бы видеть приоритет стратегий формата `split2>disorder2>fake` и тд. Ну и с инфой почему, как это описано для wssize

И кстати, почему blockcheck не проверяет варианты с `--oob`, у меня с ним тоже работает

**Comment by numbereleven-a** [comment]
Приоритет > стараться уложиться в fake,split2,split. disorder применять только, если не работают вариант split2 или split.
--oob у меня проверяется и показывается в стратегиях для tpws

**Reply by bol-van** [reply]
Блокчек и не есть панацея, о чем он в конце предупреждает. Думать все равно надо
Приоритет более-менее расставлен.
disorder ломает,например, http на моих провайдерах. Это общее поведение ТСПУ
